<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_User</class>
 <widget class="QWidget" name="Create_User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 900 16pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>710</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Admin Page</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>1001</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Create New Account</string>
   </property>
   <widget class="QPushButton" name="Create_New_Account_PB">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Create New Account</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>41</y>
      <width>1001</width>
      <height>313</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 900 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Email_LE"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 900 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Full Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Name_LE"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 900 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Password: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PW_LE"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 900 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Age_LE"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 900 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Initial Balance:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Balance_LE"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 900 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Account Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Account_Type_LE"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
